feat(mkdocs): fix typos and errors (#4530)
scaleway/docs-content: tutorials/using-bucket-website-with-mkdocs/index.mdx

[MkDocs](https://www.mkdocs.org/) is a static site generator that allows users to generate a static website from a set of Markdown files. It is written in Python and uses Markdown for content creation.
We will use MkDocs to generate our website, and Material for MkDocs which is a documentation framework that extends MkDocs with additional features and customization options, to create visually appealing and user-friendly documentation websites.

> Find out how to Set up MkDocs and configure your website and find out how to preview your website and deploy it.

Reviewer: ```suggestion
You will learn to set up MkDocs and configure, preview, and deploy your website.
```
